File: SlimFileManager/src/main/java/com/slim/slimfilemanager/utils/Bookmark.java
package com.slim.slimfilemanager.utils;

import android.database.Cursor;

public class Bookmark {

    private String name;

    private String path;

    private int fragmentId;

    private int menuId;

    public Bookmark() {
    }

    public Bookmark(Cursor cursor) {
        setName(cursor.getString(0));
        setPath(cursor.getString(1));
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

    public int getFragmentId() {
        return fragmentId;
    }

    public void setFragmentId(int fragmentId) {
        this.fragmentId = fragmentId;
    }

    public int getMenuId() {
        return menuId;
    }

    public void setMenuId(int menuId) {
        this.menuId = menuId;
    }
}

File: manager/src/main/java/com/slim/slimfilemanager/utils/Bookmark.kt
package com.slim.slimfilemanager.utils

import android.database.Cursor

class Bookmark {

    var name: String? = null

    var path: String? = null

    var fragmentId: Int = 0

    var menuId: Int = 0

    constructor() {}

    constructor(cursor: Cursor) {
        name = cursor.getString(0)
        path = cursor.getString(1)
    }
}
